test/unified_format.py

```diff
@@ -1229,6 +1235,16 @@ def _databaseOperation_createCommandCursor(self, target, **kwargs):
 
         return cursor
 
+    def kill_all_sessions(self):
+        if getattr(self, "client", None) is None:
+            return
+        try:
+            self.client.admin.command("killAllSessions", [])
```

Comment:
In the legacy runner, we would run this on all mongoses. I think we should do that here too.

Reply:
I couldn't find the equivalent of `mongos_clients`, it looks like all clients go through this line: https://github.com/mongodb/mongo-python-driver/blob/f757fe39cc12520e299090adf750fc07eb03dec8/test/unified_format.py#L501

Reply:
We'll need to create those clients like is done in the legacy txn tests: https://github.com/mongodb/mongo-python-driver/blob/992d1507e7a49292b555cc0201d95d49234c6228/test/test_transactions.py#L71

Reply:
Done